tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -220,147 +145,172 @@ def __call__(self, op):
     return device_name
 
 
-def _create_device_setter(is_cpu_ps, worker, num_gpus):
+def _create_device_setter(avg_on_gpu, worker, num_gpus):
```

Comment:
I looked at that before and I could not see where it was passing in the possible ps_servers to use for the balancing.  I am fine with bytes, that might even work better depending on the situation.  If it works I am ok simplifying my examples and the code